client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -123,24 +230,21 @@ define([
                     };
                 }
                 // return the post saved
-                return operation().then(function (post) {
-                    $rootScope.$broadcast('lb.posts.updated');
-                    // post here doesn't contain the items...
-                    $rootScope.$broadcast('lb.posts.saved', post);
-                    return post;
-                });
+                return operation();
             });
         }
 
         function removePost(post) {
-            return api.posts.remove(post).then(function() {
-                $rootScope.$broadcast('lb.posts.updated');
-                $rootScope.$broadcast('lb.posts.removed', post);
-            });
+            return api.posts.remove(post);
         }
 
         return {
+            posts: posts,
```

Comment:
`posts` is the reference of the `getPosts` method, 
so actually the `posts` is just a link and `getPosts`  is the old method.
if is confusing ... I can rename to `requirePosts` or `fetchPosts`